bpf/lib/bpf_cgroup.h

```diff
@@ -391,4 +391,64 @@ __init_cgrp_tracking_val_heap(struct cgroup *cgrp, cgroup_state state)
 	return heap;
 }
 
+/**
+ * __set_task_cgrpid_tracker() Sets the cgrpid_tracker of a task.
+ * It reads tetragon_conf if not available then exit.
+ * If tetragon_conf is available then checks if the task
+ * execve_map_value->cgrpid_tracker is set, if so do nothing.
+ * If not set then fetch task cgroup and cgroup level, compare it against
```

Comment:
This part also seems outdated, since there is no comparison with the cgroup level.

Reply:
Yeh due to the split.